apps/frontend/src/app/3_organisms/Footer/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from 'react';
+
+import { Link, Footer as UIFooter } from '@sovryn/ui';
+
+import { SocialLinks } from '../../2_molecules';
+import { SovrynLogo } from '../../2_molecules/SovrynLogo/SovrynLogo';
+
+export const Footer: FC = () => {
+  const links = [
+    {
+      id: 'start',
+      href: 'https://wiki.sovryn.app/en/getting-started',
+      name: 'Getting started',
+    },
+    {
+      id: 'alfa',
+      href: 'https://live.sovryn.app/',
+      name: 'Sovryn Alpha',
+    },
+    {
+      id: 'blog',
+      href: 'https://www.sovryn.app/blog',
+      name: 'Blog',
+    },
+    {
+      id: 'fee',
+      href: '#',
+      name: 'Fee prices',
+    },
+    {
+      id: 'terms',
+      href: '#',
+      name: 'Terms & Conditions',
+    },
+    {
+      id: 'policy',
+      href: '#',
+      name: 'Privacy policy',
+    },
+  ];
+
+  return (
+    <UIFooter
+      leftContent={
+        <SovrynLogo
+          dataAttribute="logo"
```

Comment:
Pass in "Powered by Bitcoin" text and small Sovryn logo (without "Sovryn" text) here, from Figma design